test/e2e/events_test.go

```diff
@@ -0,0 +1,337 @@
+package e2e
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	eventv1beta1 "k8s.io/api/events/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+
+	"github.com/KohlsTechnology/eunomia/pkg/apis"
+	gitopsv1alpha1 "github.com/KohlsTechnology/eunomia/pkg/apis/eunomia/v1alpha1"
+)
+
+// TestJobEvents_JobSuccess verifies that a JobSucceeded event is emitted by
+// eunomia after a simple GitOpsConfig is executed.
+func TestJobEvents_JobSuccess(t *testing.T) {
+	ctx := framework.NewTestCtx(t)
+	defer ctx.Cleanup()
+
+	namespace, err := ctx.GetNamespace()
+	if err != nil {
+		t.Fatalf("could not get namespace: %v", err)
+	}
+	defer DumpJobsLogsOnError(t, framework.Global, namespace)
+	err = framework.AddToFrameworkScheme(apis.AddToScheme, &gitopsv1alpha1.GitOpsConfigList{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	eunomiaURI, found := os.LookupEnv("EUNOMIA_URI")
+	if !found {
+		eunomiaURI = "https://github.com/kohlstechnology/eunomia"
+	}
+	eunomiaRef, found := os.LookupEnv("EUNOMIA_REF")
+	if !found {
+		eunomiaRef = "master"
+	}
```

Comment:
Probably not a task for this PR, but I think this block could be moved into an "initialize test" function because it seems to be common for every test. @seanmalloy / @akavel if you think this is a good idea I could create the issue for it.

Reply:
I'm totally for it, was thinking about it too, unfortunately just didn't have enough time to fit it in yet. If you could create an issue, that would be awesome; I have some particular ideas how to try and approach this for some extra Go street cred, I'll try to list them as a comment then. Unless I'll find time to do it myself anyway before you open the issue ;P hm, actually, I think I'll try to resist, and leave it as a cookie for grabs for someone else, for bonus Go tutoring value :) unless everybody gets too scared to touch it for fear of an annoying review ;P

Reply:
I'm totally for it, was thinking about it too, unfortunately just didn't have enough time to fit it in yet. If you could create an issue, that would be awesome; I have some particular ideas how to try and approach this for some extra Go street cred, I'll try to list them as a comment then. Unless I'll find time to do it myself anyway before you open the issue ;P hm, actually, I think I'll try to resist, and leave it as a cookie for grabs for someone else, for bonus Go tutoring value :) unless everybody gets too scared to touch it for fear of an annoying review ;P

Reply:
Well done, it looks like you managed to contain your excitement long enough for me to create #214 😄 Any feedback/suggestions on it are greatly appreciated as always!